features/miniAlert.js

```diff
@@ -0,0 +1,25 @@
+//Thanks Vals
+function inDungeons() {

```

Comment:
should use an arrow function since you don't need the arguments

Reply:
Didn't Ask.